src/anomalib/models/image/glass/backbones.py

```diff
@@ -0,0 +1,50 @@
+import torchvision.models as models
```

Comment:
It would be nice to attribute files like these to the original implementation. Also, include an entry in `third-party.txt`
https://github.com/open-edge-platform/anomalib/blob/main/third-party-programs.txt